src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -282,5 +360,20 @@ object SbtSlamData extends AutoPlugin {
         currentVersion + "-" + "git rev-parse HEAD".!!.substring(0, 7)
       else
         currentVersion
-    })
+    },
+
+    Test / test := (Def.taskDyn {
```

Comment:
I think a better approach would be to do this same thing in `update` rather than `test`. There's no reason to wait for `test`, since that's going to happen *after* `compile`, whereas `update` happens right at the start.

Reply:
Well, I was just (blindly) following your suggestion. That said, I did look into it before PRing it, and `update` is a task and a dependency on a lot of other tasks, so if I placed it there, it would keep repeating these things over and over. I'd have to turn `versionCheck` into a value, for it to get cached and avoid starting an external sbt every time, and I'n not sure it can be done. I think there were restrictions on what could be used inside a `taskDyn`. I'll try.

Reply:
It just doesn't work out as a setting -- it gets into an infinite loop. I'm not sure how I'd break out of that one. Doing this in `update` but as a task would probably have the same issue, in addition to the performance issue. I think this is the "good enough" scenario.

Also, even if it fails it does run all the tests, which can be useful.